RocketChat-Stats.xml

```diff
@@ -19,7 +19,7 @@
             </groups>
             <applications>
                 <application>
-                    <name>RocketChat</name>
+                    <name>Custom - Service - Rocketchat</name>
```

Comment:
You propose to change application name "RocketChat" with "Custom - service - RocketChat". Is this a Zabbix convention ? Is there a convention ? it's a very good idea, but if every contributor propose own convention, it's a wild.

However, I will adopt this convention for our monitoring system, so don't change anything. At this time, we haven't convention and it's wild ;)

Reply:
Just rename it as you prefer - thats our internal convention which we use to distinguish internally modified templates.